src/main/kotlin/com/cognifide/gradle/aem/instance/tail/LogTailer.kt

```diff
@@ -1,16 +1,19 @@
 package com.cognifide.gradle.aem.instance.tail
 
+import com.cognifide.gradle.aem.instance.tail.io.ConsolePrinter
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.channels.SendChannel
 import kotlinx.coroutines.launch
 
 class LogTailer(
     private val source: LogSource,
     private val destination: LogDestination,
-    private val logsAnalyzerChannel: SendChannel<Log>? = null
+    instance: InstanceLoggingInfo = InstanceLoggingInfo.default(),
+    private val logsAnalyzerChannel: SendChannel<Log>? = null,
+    private val printer: ConsolePrinter = ConsolePrinter.devNull()
```

Comment:
`devNull` ? :) make it not related with unix os :D more generic...

Reply:
e.g `ConsolePrinter.none()` or `ConsolePrinter.off()`

Reply:
OK